.travis.yml

```diff
@@ -1,4 +1,7 @@
+dist: trusty
```

Comment:
a different dist is needed to pass the env issue travis is having with oraclejdk8